backend/GeometryParser.cpp

```diff
@@ -0,0 +1,128 @@
+#include "GeometryParser.h"
+#include <iostream>
+
+GeometryParser::GeometryParser () {
+    this->objectNames = {"point", "circle", "line", "triangle", "square", "rectangle"};
+    this->functionNames = {"angle", "intersection"};
+}
+
+std::string GeometryParser::identify (std::string s) {
```

Comment:
I noticed we only accept lowercase input from the user. Maybe we could make it a little more generic by first lowercasing s and then comparing it?